### DISCUSSION TITLE
Multi Monitor - Looking forward to see successor of ac'tiv Aid

### DISCUSSION THREAD

**Discussion by greg4b** [ewerybody/a2#227] [General]
_I love to see that ac'tiv Aid gets a successor!_

Happy that "**hot strings**" is useable and looking forward for "**Multi Monitor**" - ability to define short-cuts for putting windows to certain monitors.

Thank you,

Gregor

**Comment by ewerybody** [comment]
Hey @greg4b! Yeah, Thank you! There are still improvements to be done on HotStrings. But that should goto the .. Oh! I already made an issue on that [#23](https://github.com/ewerybody/a2.modules/issues/23) 😄.

But what's Multi Monitor? Ah this one?
![image](https://user-images.githubusercontent.com/218956/107499987-ecad1f80-6b95-11eb-9b19-728d60667b2f.png)

I recently put in some library stuff from the Autohotkey community. See [mdmf.ahk](https://github.com/ewerybody/a2/blob/master/lib/Autohotkey/lib/mdmf.ahk) ... so this might be easy to implement.

I'd actually try that with [the new **a2.modlab** repository](https://github.com/a2script/a2.modlab). It doesn't have a release yet. But this might be a good starting point.

**Reply by Moosbueffel** [reply]
@greg4b I also use 2 monitors and have a well message for you. Perhasp it is what you are looking for. Have a look at [powertoys ](https://docs.microsoft.com/en-us/windows/powertoys/fancyzones) from MS and check FancZones. You will be happy, if you like to define monitor zones.

Also recommended the [WIN10VirtualDesktopEnhancer](https://github.com/sdias/win-10-virtual-desktop-enhancer) for movinw windows, define background ,... if you use virtual desktops.

BTW @ewerybody this is also an ahk. Perhaps it is possible ti integrate in a2??

**Reply by greg4b** [reply]
Yes that is "Multi Monitor" - simply define shortcut's to move windows directly to desired monitor.

Also "Mouse Clip" is one of my most used too from ac'tiv Aid: Select text and paste it with middle mouse click (no ctrl+c and no ctrl+v)

@Moosbueffel thanks for your reply - I've noticed powertoys before. But since I'm on a "locked down" computer (no admin rights), I can't go with that (they provide no portable power toys).  I do not use the Virtual Desktop currently.
